src/index.js

```diff
@@ -6,3 +6,33 @@ import "./index.scss";
 
 // install snippets styles
 import "highlight.js/styles/default.css";
+
+window.onload = () => {
+  const localeSeletor = document.getElementById("localeSelector");
+  localeSeletor.innerHTML = isZhCN() ? 'English' : "中文";
```

Comment:
I think `textContent` is the right property to set here.

```suggestion
  localeSeletor. textContent = isZhCN() ? 'English' : "中文";
```

Once #449 is merged, I think we'll want to replace these hard coded strings with strings loaded from the locale files.

Reply:
The `textContent` change is good, but beware changing the locale names based on the current locale. If the UI is in Chinese and you only speak English, you would not want to find the menu item for English if it's written with Chinese characters.

Reply:
Is using flag emojis a way to solve this issue?

Reply:
Flags are even worse, because lots of countries have multiple very popular languages (Canada, India, China, etc.). In a language selector, write out the language names in the language spoken by the language being described.

Reply:
OK, does that mean a dropdown like this in the navbar is more appropriate?

| 🗣 |
| -------|
| English |
| 中文 |

~This changes the selector to use _putonghua_ (Mandarin) rather than _zhongwen_ (Chinese nationality).~ (whoops I just checked with my wife. I was getting confused between _zhongwen_ and _zhongguoren_).

Or maybe like:

```
| ---------- |
| 🗣 English |
| ---------- | (hidden in dropdown)
|    中文    |
| ---------- |
```

Reply:
Google has an icon for language that's free to use, iirc. That's pretty well established if you want to hide it in a drop-down

Reply:
I think hide it in a dropdown is more appropriate for multi languages. It's easy to switch if there's only two language for now.